mcc/odin/odin/snapshot_list_test.go

```diff
@@ -2,25 +2,41 @@ package odin_test
 
 import (
 	"testing"
+	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/rds"
 
 	"github.com/poka-yoke/spaceflight/mcc/odin/odin"
 )
 
+const (
+	RFC8601 = "2016-01-02T15:04:05+07:00"
```

Comment:
Oh! This is wrong, it should not be 2016, but 2006... Besides this, the way Go language designers thought is pretty different. They take a date defined by the number 1 to 7 (1 2 3 4 5 6 7), where each digit is month, day, hour, minute, second, year after 2000, and timezone. The format is defined with that date and time.
What I don't understand now is why the tests where passing.